test/topology_experimental_raft/test_mv_tablets.py

```diff
@@ -176,29 +187,29 @@ async def inject_error_on(manager, error_name, servers):
 # separate thread) ourselves.
 @pytest.mark.asyncio
 @skip_mode('release', 'error injections are not supported in release mode')
-@skip_mode('debug', '10-node test is too slow in debug mode')
 async def test_tablet_alternator_lsi_consistency(manager: ManagerClient):
     """A reproducer for a bug where Alternator LSI was not using synchronous
        view updates when tablets are enabled, which could cause strongly-
        consistent read of the LSI to miss the data just written to the base.
-       We use a fairly large cluster to increase the chance that the tablet
-       randomization results in non-local pairing of base and view tablets.
-       We could make this test fail more reliably and only need 4 nodes if
-       we had an API to control the tablet placement.
-       We also use the "delay_before_remote_view_update" injection point
-       to add a delay to the view update - without it it's almost impossible
-       to reproduce this issue on a fast machine.
+
+       We use a cluster of just two nodes and RF=1, and control the tablets
+       so all base tablets will be in node 0 and all view tablets will be
+       in node 1, to ensure that the view update is remote and therefore
```

Comment:
This won't work once we fix local indexes, as local indexes will be co-located with the table even when using tablets.

Reply:
The test is is currently written will *force* the view's tablets to be moved to a different node, so if "co-location" will forbid this, the test will fail clearly. If it won't be forbidden to explicitly move the tablets, the test will continue to work, but arguably won't really be an interesting regression test because it tests a non-realistic scenario. I'm not sure exactly what to do. I guess that when we do this "co-location" thing we'll need to add more tests.

Reply:
It will be forbidden.